src/Symfony/Component/Intl/Resources/data/locales/af.json

```diff
@@ -94,6 +95,8 @@
         "el_CY": "Grieks (Siprus)",
         "el_GR": "Grieks (Griekeland)",
         "en": "Engels",
+        "en_001": "Engels (Wêreld)",
```

Comment:
from a user perspective, this creates 2 entries in the Locale form type. It may be confusing :thinking: but i tend to believe it should be filtered manually, if desired.

Reply:
so the user may select en_001 to explicitly provide a region and opt out from some specific region (hence "world"), whereas in case of  "en" the app may assume any region.

Fine as-is for me :+1: